src/test/java/com/lmax/disruptor/dsl/DisruptorTest.java

```diff
@@ -86,6 +86,21 @@ public void tearDown() throws Exception
         executor.joinAllThreads();
     }
 
+    @Test
+    public void shouldHaveStartedAfterStartCalled() throws Exception {
+      if (disruptor.hasStarted())
```

Comment:
Using `if` and `fail` is an _interesting_ way of writing this test.

Would it not be simpler use `assertFalse(disruptor.hasStarted(), "Should only be set to started after start is called");`?

Reply:
ah yeah! much cleaner! I cribbed from the test below and thought it was notably like golang